app/src/main/res/layout/data_storage_location_item.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              android:layout_width="match_parent"
+              android:layout_height="wrap_content"
+              android:orientation="horizontal"
+              android:paddingLeft="16dp"
+              android:paddingRight="16dp"
+              android:minHeight="?android:attr/listPreferredItemHeight">
+
+  <LinearLayout
```

Comment:
kill weights, make it `alignParentStart` and `alignParentLeft`, and make it `toLeftOf` the box, which is `alignParentRight` and `alignParentEnd`.